test/examples/search/warnings.ts

```diff
@@ -51,10 +55,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await es.indices.addBlock({ index: testIndex, block: 'write' });
       try {
         log.info(`rolling up ${testIndex} index...`);
-        await es.rollup.rollup({
-          index: testIndex,
-          rollup_index: testRollupIndex,
-          config: { fixed_interval: '1h' },
+        // es client currently does not have method for downsample
```

Comment:
this had to change in this PR because of https://github.com/elastic/elasticsearch/pull/89809

Reply:
Great! Could that the test was already skipped :D